### ISSUE TITLE
Axis to Int and Axis to Float don't work with Text Splitter

### ISSUE THREAD

**Issue by geroldmeisinger** [kenjiqq/qq-nodes-comfyui#17]
using "Axis to Int" in steps
```
KSampler

linspace() received an invalid combination of arguments - got (Tensor, Tensor, str), but expected one of:
 * (Tensor start, Tensor end, int steps, *, Tensor out = None, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
 * (Number start, Tensor end, int steps, *, Tensor out = None, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
 * (Tensor start, Number end, int steps, *, Tensor out = None, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
 * (Number start, Number end, int steps, *, Tensor out = None, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
```

using "Axis to Float" in cfg
```
KSampler

must be real number, not str
```

![Image](https://github.com/user-attachments/assets/01543bf6-f7f1-4359-848c-76b1f1c504aa)

[workflow.json](https://github.com/user-attachments/files/19429194/qq-nodes.json)

**Comment by geroldmeisinger** [comment]
https://github.com/kenjiqq/qq-nodes-comfyui/blob/main/nodes.py#L472

```
def create_axis_class(name):
    class_dict = {
        'RETURN_TYPES': (name,),
    }

    return type(f"AxisTo{name}", (AxisBase,), class_dict)
```

you're just casting, not converting

**Comment by geroldmeisinger** [comment]
proposal:
```
def create_axis_class(name):
    id = lambda x: x
    funcs = {
        "STRING": str,
        "MODEL": id,
        "INT": int,
        "NUMBER": float,
        "FLOAT": float,
        "BOOLEAN": lambda x: bool(strtobool(x)),
        "VAE": id,
        "CLIP": id,
		}
    class_dict = {
        'RETURN_TYPES': (name,),
        'run': lambda self, axis: (funcs[name](axis), )
    }

    return type(f"AxisTo{name}", (AxisBase,), class_dict)

```

**Comment by kenjiqq** [comment]
The point of these nodes was never really to actually cast the values, they were a workaround to allow you to connect the output of the grid helper to any other nodes input. But that was before i figured out how you make wildcard types and made the AxisToAny node, after that i've just been using that for everything.
So i have actually been meaning to just remove all AxisTo nodes and make the helper directly output wildcard types so it can connect to anything.
And if you need to cast anything you should just use nodes specifically made for that
